dotCMS/src/main/java/com/dotmarketing/servlets/taillog/TailLogServlet.java

```diff
@@ -132,12 +129,17 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
 		try {
 			while (thread.isAlive()) {
                 String write = listener.getOut(true).toString();
-                response.getOutputStream().print(write);
-                response.getOutputStream().print("<script>doS();</script>");
-                response.getOutputStream().flush();
+				if (UtilMethods.isSet(write)) {
+					response.getOutputStream().print(write);
+					response.getOutputStream().print("<script>doS('" + write + "');</script>");
+					response.getOutputStream().flush();
+				}
```

Comment:
Keep in mind this is a long polling servlet - we still need it to output something and send it over the wire to keep the http request alive, even if we ignore the output on the screen.

Reply:
yes, BE code reverted

Reply:
@wezell are not going to change the approach for this?